src/__tests__/core/cardanoGetAddress.spec.js

```diff
@@ -1,51 +1,367 @@
 /* @flow */
+import {
+    CARDANO_ADDRESS_TYPE,
+    HARDENED,
```

Comment:
does `HARDENED` need to be declared in constants since the only usage is only here in one test case? 
wouldn't be better to use `path: [0x80000000 + 49]` in line 29 ?

Reply:
Makes sense. Removed [here](https://github.com/trezor/connect/pull/638/commits/a272ece99493e82407d8fa6e283ab76990ea07ab).